.github/CODEOWNERS

```diff
@@ -49,6 +49,7 @@
 
 ## Mobile Platform Team
 /packages/app-metadata-controller @MetaMask/mobile-platform
+/packages/analytics-controller    @MetaMask/mobile-platform
```

Comment:
If this controller is supposed to be shared across clients shouldn't we also add extension platform ?